packages/plugin-solana/src/index.ts

```diff
@@ -12,6 +12,12 @@ import { Plugin } from "@ai16z/eliza";
 import { walletProvider } from "./providers/wallet.ts";
 import { trustScoreProvider } from "./providers/trustScoreProvider.ts";
 import { trustEvaluator } from "./evaluators/trust.ts";
+import { TokenProvider } from "./providers/token.ts";
+import { WalletProvider } from "./providers/wallet.ts";
+import { Connection } from "@solana/web3.js";
+import { PublicKey } from "@solana/web3.js";
+
+export { TokenProvider, WalletProvider, Connection, PublicKey };
```

Comment:
remove the solana exports and import where needed from the packages

Reply:
Good catch, this was left over from another test I was working on, but ended up simplifying for this first PR